zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -563,6 +569,12 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/UserTaskSearchQueryResponse"
+            application/vnd.camunda.api.keys.number+json:
+              schema:
+                $ref: "#/components/schemas/UserTaskSearchQueryResponse"
+            application/vnd.camunda.api.keys.string+json:
+              schema:
+                $ref: "#/components/schemas/UserTaskSearchQueryResult"
```

Comment:
💭 Just something I noticed but is a follow-up: we don't do this media type handling for the search requests, only for the responses. The requests can contain filters on key attributes as well, though. Should we introduce this there as well, at least on OpenAPI-level? I don't think we have to adjust the REST controllers in any way. But for users it needs to be clear they can also send keys as Strings, right?

Reply:
Yes, I guess those need adjustment as well.
I'll create a follow-up ticket for it.